docs/getting-started.md

```diff
@@ -511,42 +511,101 @@ const messageForClaimer = new Kilt.Message(
+They may choose to create a copy disclosing only selected properties. In the following example, we only disclose the `name` attribute of our credential, omitting `age`:
 
 ```typescript
-const copiedCredential = Kilt.AttestedClaim.fromAttestedClaim(
-  JSON.parse(JSON.stringify(myAttestedClaim))
-)
-copiedCredential.request.removeClaimProperties(['age'])
+const credentialCopy = myAttestedClaim.createPresentation(['name'])
 
 const messageBodyForVerifier: MessageBody = {
-  content: [copiedCredential],
+  content: [credentialCopy],
   type: Kilt.Message.BodyType.SUBMIT_CLAIMS_FOR_CTYPES,
 }
 const messageForVerifier = new Kilt.Message(
   messageBodyForVerifier,
   claimer.getPublicIdentity(),
   verifier.getPublicIdentity()
 )
+
+const encrypted = messageForVerifier.encrypt(
+  claimer,
+  verifier.getPublicIdentity()
+)
 ```
 
 ### 6.2. Verify presentation
 
-When verifying the claimer's message, the verifier has to use their session which was created during the CTYPE request.
-The result will be a boolean indicating the result of the verification and the attested claim(s) which are either sent in their entirety OR have been stripped off of the properties that the verifier did not request to verify.
+When verifying the claimer's message, the verifier checks that 3 important requirements are met:
+
+- The credential is valid, i.e. it has been registered to the public ledger, has not been revoked, and the data has not been tampered with.
+- The attester who registered the credential is one that this verifier trusts.
+- The submission is coming from the legitimate owner of this credential, i.e. the credential has not been stolen.
 
 ```typescript
+const messageForVerifier = Kilt.Message.decrypt(encrypted, verifier)
+// Our messages are signed, so we authenticate the sender already during decryption, which internally calls:
+Kilt.Message.ensureHashAndSignature(encrypted, claimer.address)
+Kilt.Message.ensureOwnerIsSender(messageForVerifier)
+
 if (
   messageForVerifier.body.type ===
   Kilt.Message.BodyType.SUBMIT_CLAIMS_FOR_CTYPES
 ) {
   const claims = messageForVerifier.body.content
-  const isValid = await Kilt.AttestedClaim.fromAttestedClaim(claims[0]).verify()
-  console.log('Verifcation success?', isValid)
   console.log('Attested claims from verifier perspective:\n', claims)
+
+  const credential = Kilt.AttestedClaim.fromAttestedClaim(claims[0])
+
+  // credential is registered on chain, has not been tampered with, and has not been revoked
+  const isValid = await credential.verify()
+  console.log('Verification success?', isValid)
+
+  // AttestedClaim exposes attester address to compare against a whitelist of trusted attesters
+  console.log('Attester address:', credential)
 }
 ```
 
+### 6.3. Replay Protection
+
+In certain use cases, an attacker may intercept and copy credential submissions traveling from claimers to verifiers in an attempt to convince the verifier to accept the credential submission again later on.
```

Comment:
Or, the claimer copies the message to give it to their friends.

Reply:
yeaaaah, but... if the owner is in on it, they could also just re-submit. It's not like you would alter the origin/sender by giving someone a copy of your message